common/changes/@microsoft/load-themed-styles/es6_2019-12-17-05-40.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/load-themed-styles",
+      "comment": "adds es6 library for load-theme-styles",
```

Comment:
```suggestion
      "comment": "Add a \"module\" entry in package.json for the es6-style library.",
```

Reply:
@kenotron I tried to accept this change, but it seems your branch is protected against commits from maintainers.

Reply:
![image](https://user-images.githubusercontent.com/34725/71328505-0a6c8080-24cd-11ea-9a75-5e88afea9899.png)

Reply:
I'll just accept ian's suggestion real quick!